platform/src/main/java/org/stellar/anchor/platform/payment/observer/stellar/StellarPaymentObserver.java

```diff
@@ -324,5 +386,10 @@ class StreamHealth {
   boolean threadTerminated;
 
   boolean stopped;
+
+  @SerializedName("last_event_id")
   String lastEventId;
+
+  @SerializedName("silence_duration_seconds")
+  String silenceDurationSeconds;
```

Comment:
`silenceDurationSeconds` is kinda confusing. What does silence mean?

seconds_since_last_event would be more descriptive.

Reply:
renamed